src/test/emulators/fixtures.ts

```diff
@@ -0,0 +1,118 @@
+import { findModuleRoot, FunctionsRuntimeBundle } from "../../emulator/functionsEmulatorShared";
+
+export const TIMEOUT_LONG = 10000;
+export const TIMEOUT_MED = 5000;
+
+const cwd = findModuleRoot("firebase-tools", __dirname);
+export const FunctionRuntimeBundles = {
```

Comment:
As a JS impostor, can you explain this file to me?

Reply:
A [fixture](https://github.com/junit-team/junit4/wiki/test-fixtures) is a hard-coded object used for tests. Since we now need these FunctionRuntimeBundles in the new hub tests, I felt it made sense to move these shared objects into a joint file for ease of use.

Reply:
SG